src/client/utils/download/download.ts

```diff
@@ -36,7 +36,7 @@ export function getMIMEType(fileType: FileFormat) {
 function saveFile(part: string | Buffer, fileName: string, fileFormat: FileFormat, fileEncoding: string) {
   const type = `${getMIMEType(fileFormat)};charset=${fileEncoding}`;
   const blob = new Blob([part], { type });
-  fileSaver.saveAs(blob, `${fileName}.${fileFormat}`, true); // true == disable auto BOM
+  fileSaver.saveAs(blob, `${fileName}.${fileFormat}`, true);
```

Comment:
I removed this comment because that's what last parameter name is. Comment seems redundant.

Reply:
I do not see parameter name here.